sceptre/cli/drift.py

```diff
@@ -92,15 +95,14 @@ def drift_show(ctx, path):
     )
 
     plan = SceptrePlan(context)
-    responses = plan.drift_show()
+    responses = plan.drift_show(drifted)
 
     output_format = "json" if context.output_format == "json" else "yaml"
 
     exit_status = 0
     for stack, (status, response) in responses.items():
         if status in BAD_STATUSES:
             exit_status += 1
-        response.pop("ResponseMetadata", None)
```

Comment:
Why are you removing this?

Reply:
@jfalkenstein Because I had this idea that perhaps all of this 'filtering' should occur inside a single, overcomplicated filtering function!